imagery/i.biomass/testsuite/test_i_biomass.py

```diff
@@ -0,0 +1,307 @@
+    def setUpClass(cls):
+        """Set up input rasters and configure test environment."""
+        cls.use_temp_region()
+        cls.runModule("g.region", n=10, s=0, e=10, w=0, rows=10, cols=10)
+        cls._create_input_rasters()
+
+    @classmethod
+    def _create_input_rasters(cls):
+        """Helper method to create all input rasters with defined expressions."""
+        expressions = {
+            "fpar": "col() * 0.1",
+            "lightuse_eff": "row() * 0.1",
+            "latitude": "45.0",
+            "dayofyear": "150",
+            "transmissivity": "0.75",
+            "water": "0.8",
+        }
+        for raster_key, expr in expressions.items():
+            raster_name = cls.input_rasters[raster_key]
+            cls.runModule(
+                "r.mapcalc", expression=f"{raster_name} = {expr}", overwrite=True
+            )
+
+    @classmethod
+    def tearDownClass(cls):
+        """Clean up generated data and reset the region."""
+        rasters_to_remove = list(cls.input_rasters.values()) + [cls.output_raster]
+        cls.runModule(
+            "g.remove",
+            type="raster",
+            name=",".join(rasters_to_remove),
+            flags="f",
+            quiet=True,
+        )
+        cls.del_temp_region()
+
+    def test_biomass_against_reference(self):
+        """
+        Compare output of i.biomass with precomputed reference statistics.
+        The reference raster was created using the following commands:
+        g.region n=10 s=0 e=10 w=0 rows=10 cols=10
+        r.mapcalc "test_fpar = col() * 0.1"
+        r.mapcalc "test_lightuse_eff = row() * 0.1"
+        r.mapcalc "test_latitude = 45.0"
+        r.mapcalc "test_dayofyear = 150"
+        r.mapcalc "test_transmissivity = 0.75"
+        r.mapcalc "test_water_availability = 0.8"
+        i.biomass fpar=test_fpar lightuse_efficiency=test_lightuse_eff \
+        latitude=test_latitude dayofyear=test_dayofyear \
+        transmissivity_singleway=test_transmissivity \
+        water_availability=test_water_availability output=biomass_reference
+        r.univar -g biomass_reference
+        """
+        self.assertModule(
+            "i.biomass",
+            fpar=self.input_rasters["fpar"],
+            lightuse_efficiency=self.input_rasters["lightuse_eff"],
+            latitude=self.input_rasters["latitude"],
+            dayofyear=self.input_rasters["dayofyear"],
+            transmissivity_singleway=self.input_rasters["transmissivity"],
+            water_availability=self.input_rasters["water"],
+            output=self.output_raster,
+            overwrite=True,
+        )
+        self.assertRasterExists(self.output_raster)
+
+        expected_stats = (
+            "min=4.51592674628609\n"
+            "max=75.2654457714349\n"
+            "mean=33.1167961394313\n"
+            "stddev=18.5128518411928"
+        )
+
+        self.assertRasterFitsUnivar(
+            raster=self.output_raster, reference=expected_stats, precision=1e-8
```

Comment:
In other tests we use slightly lower precision which is sufficient. Also you can reduce the number of decimal places in the stats.
```suggestion
            raster=self.output_raster, reference=expected_stats, precision=1e-6
```

Reply:
I have changed to six decimal places and also reduced the number of decimal places in the stats.